contracts/adapters/Configuration.sol

```diff
@@ -82,13 +81,13 @@ contract ConfigurationContract is IConfiguration, DaoConstants, MemberGuard {
 
     function sponsorProposal(
         DaoRegistry dao,
-        uint256 _proposalId,
+        uint64 proposalId,
```

Comment:
if it's a parameter, you need to keep it as uin256, check that it isn't too big and then cast it into uint64. Otherwise there is a risk of overflow